plugins/module_utils/foreman_helper.py

```diff
@@ -1399,11 +1431,9 @@ class KatelloAnsibleModule(KatelloMixin, ForemanAnsibleModule):
     pass
 
 
-class KatelloEntityAnsibleModule(KatelloMixin, ForemanEntityAnsibleModule):
+class KatelloScopedMixin(KatelloMixin):
```

Comment:
I wonder if we really need this, or if the scope enforcement should happen in `KatelloMixin` directly.

Reply:
Maybe yes, i guess, you need to look into an non entity katello module to decide.